packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewer.js

```diff
@@ -0,0 +1,157 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { withTheme } from 'styled-components'
+import { Bar } from '@vx/shape'
+import { Group } from '@vx/group'
+import { AxisBottom, AxisLeft } from '@vx/axis'
+import { scaleBand, scaleLinear } from '@vx/scale'
+import { withParentSize } from '@vx/responsive'
+import Chart from '../SVGComponents/Chart'
+import Background from '../SVGComponents/Background'
+
+const BarChartViewer = React.forwardRef(function BarChartViewer(props, ref) {
+  const {
+    barStyles: {
+      padding
+    },
+    data,
+    parentHeight,
+    parentWidth,
+    theme: { dark, global: { colors, font } },
+    xAxisLabel,
+    xAxisMargin,
+    yAxisLabel,
+    yAxisMargin
+  } = props
+
+  let axisColor = (dark) ? colors.text.dark : colors.text.light
+  // Should we put white into the theme?
+  let backgroundColor = (dark) ? colors['dark-3'] : 'white'
+  const xMax = parentWidth - xAxisMargin
+  const yMax = parentHeight - yAxisMargin
+
+  const xScale = scaleBand({
+    domain: data.map(datum => datum.label),
+    rangeRound: [0, xMax],
+    padding
+  })
+  const yScale = scaleLinear({
+    domain: [0, Math.max(...data.map(datum => datum.value))],
+    rangeRound: [yMax, 0]
+  })
+
+  const xScaleTicks = xScale.domain()
+  const yScaleTicks = yScale.domain()
+  return (
+    <Chart height={parentHeight} ref={ref} width={parentWidth}>
+      <Background fill={backgroundColor} />
+      <Group left={xAxisMargin}>
+        {data.map((datum, index) => {
+          const { color, label, value } = datum
+          const fill = color || colors.brand
+          const key = `bar-${label}`
+          const barHeight = yMax - yScale(value)
+          const barWidth = xScale.bandwidth()
+          const x = xScale(label)
+          const y = yMax - barHeight
+          return (
+            <Bar
```

Comment:
Should these bars have alt text, like the bars in the Daily Classifications Chart?

Reply:
Yes, that's one of the enhancements I'd like to do after this initial PR